subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/hived_algorithm.go

```diff
@@ -869,6 +869,11 @@ func generatePodScheduleResult(
 	vc api.VirtualClusterName,
 	pod *core.Pod) internal.PodScheduleResult {
 
+	var allSuggestedNodes []string
+	for node := range suggestedNodeSet.Items() {
+		allSuggestedNodes = append(allSuggestedNodes, node.(string))
+	}
+	klog.Infof("All suggested nodes: %v", strings.Join(allSuggestedNodes, ", "))
```

Comment:
>klog.Infof( [](start = 1, length = 11)

klog.V(4).Infof(...)

Reply:
Use klog.V(4) for too verbose debug info, it can be masked by args --v=. [https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)

---
In reply to: [390086189](https://github.com/microsoft/pai/pull/4262#discussion_r390086189) [](ancestors = 390086189)

Reply:
Also log the context, like internal.Key(pod)

---
In reply to: [390086189](https://github.com/microsoft/pai/pull/4262#discussion_r390086189) [](ancestors = 390086189)